core/src/apps/ethereum/sc_constants.py

```
@@ -44,14 +48,255 @@
+        self.label = label
+        self.format = format
+        self.threshold = threshold
+
+
+class FieldType:
+    def parse(self, arg: memoryview, field_format: FieldFormat) -> int | bytes | None:
+        raise NotImplementedError
+
+
+class AddressFieldType(FieldType):
+    def parse(self, arg: memoryview, field_format: FieldFormat) -> int | bytes | None:
```

Ruff (0.14.14)

[warning] Unused method argument: `field_format`

(ARG002)